### DISCUSSION TITLE
I want to receive license key again but I couldn't

### DISCUSSION THREAD

**Discussion by P9Mew** [aclap-dev/video-downloadhelper#581] [License & Purchase requests]
### What's your issue?

Few years ago I purchase the premium with license key. Due to the recent version update required the license key but I cant find in my mail box anymore as it is too old. 

Please help to check
szeyih@hotmail.com
chinszeyih@gmail.com

### The 10 first digits (only 10) of your license number or email address

szeyih@hotmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry I only found a monthly subscription which ended 3 years ago.
